frontend/src/pages/Q&A/Q&A.jsx

```diff
@@ -99,44 +164,80 @@ function Ques(props) {
     });
     if (isValid && formdata.tags.length !== 0) {
       setIsUploadingData(true);
-      uploadData(formdata, setToast)
-        .then(() => {
-          setIsUploadingData(false);
-          setFormData({
-            title: "",
-            description: "",
-            tags: [],
-          });
-          setFormErrors({});
-          setCheckedState(new Array(Tags.length).fill(false));
-        })
-        .catch(() => {
-          setIsUploadingData(false);
-        });
+      uploadData(formdata);
     }
   };
+  function ActiveButton() {
+    setButtonPressed(!isButtonPressed);
+  }
+  function DeactiveButton() {
+    setButtonPressed(!isButtonPressed);
+  }
 
   const [getQuestions, setQuestions] = useState([]);
 
-  const fetchQuestions = () => {
-    getAllQuestion(setToast).then((data) => {
-      setLoading(false);
-      setQuestions(data);
+  function getQues() {
```

Comment:
these all should come from service folder na, i think during conflict resolved these got messed up, can u raise a fresh pr with just your changes on top of master?

Reply:
ok i am raising a fresh pr